.dockerignore

```diff
@@ -48,4 +48,8 @@ dmypy.json
 
 # docker bits
 Dockerfile*
-docker-compose*
\ No newline at end of file
+docker-compose*
+
+# Temp files
+tmp/
```

Comment:
These are generated by the comprehensive query testing, or any output files can go here